src/util/irep_ids.def

```diff
@@ -670,6 +670,7 @@ IREP_ID_ONE(bits_per_byte)
 IREP_ID_TWO(C_abstract, #abstract)
 IREP_ID_ONE(synthetic)
 IREP_ID_ONE(interface)
+IREP_ID_TWO(C_must_not_throw, #must_not_throw)
```

Comment:
For my own knowledge: What does `C` stand for?

Reply:
'comment', I suppose.